### PR TITLE
Updated filters :)

### PR DESCRIPTION
**Pull request by mmitis**

### CODE REVIEW

File: src/components/Workshops/helpers.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  sortType: WorkshopSortType,
+  isOld: boolean
+) => {
+  if (sortType === WorkshopSortType.ByPrice) {
+    return events.sort((a, b) => a.price - b.price);
+  }
+  if (isOld) {
+    return events
+      .sort((a, b) => dayjs(a.startDate).unix() - dayjs(b.startDate).unix())
+      .reverse();
```
Comment (piotrski) [inline src/components/Workshops/helpers.ts:16]: I'd just switch the sort direction and remove reverse ()
```suggestion
      .sort((a, b) =>  dayjs(b.startDate).unix() - dayjs(a.startDate).unix())
```

File: src/components/WorkshopTagButton/index.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+};
+
+export const WorkshopTagButton = ({ onClick, name, isActive }: Props) => {
+  return (
+    <button
+      type="button"
+      className={`tag__single tagsFilter ${
+        isActive ? "tag__single--active" : ""
+      }`}
+      onClick={() => onClick()}
```
Comment (piotrski) [inline src/components/WorkshopTagButton/index.tsx:16]: This arrow function is unnecessary

File: src/components/Workshops/index.tsx [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+    () =>
+      sortEvents(
+        filteredEvents.filter(
+          (event) => dayjs(event.startDate).unix() > dayjs().unix()
+        ),
+        sortType,
+        false
+      ),
+    [filteredEvents, sortType]
+  );
```
Comment (piotrski) [inline src/components/Workshops/index.tsx:89]: Most of this logic is redundant and could be moved into a helper or even custom hook.
`const { oldEvents, ongoingEvents } = sortEvents(filteredEvents, sortType)`

Comment (piotrski) [reply src/components/Workshops/index.tsx:89]: I could do a PR with that.

Comment (mmitis) [reply src/components/Workshops/index.tsx:89]: feel free to improve - i just made a draft from Tomek's code - it's not a real PR :P :P

**Pull request closed without merging**